perception/lidar_centerpoint/launch/centerpoint_vs_centerpoint-tiny/centerpoint_vs_centerpoint-tiny.launch.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="rviz_path" default="$(find-pkg-share lidar_centerpoint)/launch/centerpoint_vs_centerpoint-tiny/rviz"/>
+  <arg name="input/pointcloud" default="/sensing/lidar/concatenated/pointcloud"/>
+  <arg name="output/objects" default="objects"/>
+  <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`"/>
```

Comment:
[imo] Could you remove these 2 lines(L5-6) because they are not used in below.

Reply:
Thanks for your reminder, unneeded code is already removed from the launch file.